tests/unit/adapters/listings-adapter-test.js

```diff
@@ -2,6 +2,7 @@ import { module, test } from 'qunit';
 import ListingsAdapter from 'shopify-buy/adapters/listings-adapter';
 import version from 'shopify-buy/version';
 import Promise from 'promise';
+import 'shopify-buy/isomorphic-btoa';
```

Comment:
Should this be a concern here? Or farther up the chain? I feel like we shouldn't force the test to be aware of the isomorphs. Should we maybe instead include this behaviour in whatever file in the node tests does the equivalent of [this code]() does? https://github.com/Shopify/js-buy-sdk/blob/master/tests/index.html#L18-L22